cpp/open3d/t/geometry/PointCloud.h

```diff
@@ -422,24 +422,32 @@ class PointCloud : public Geometry, public DrawableGeometry {
 public:
     /// \brief Function to estimate point normals. If the point cloud normals
     /// exist, the estimated normals are oriented with respect to the same.
-    /// It uses KNN search if only max_nn parameter is provided, and
-    /// HybridSearch if radius parameter is also provided.
-    /// \param max_nn Neighbor search max neighbors parameter [Default = 30].
-    /// \param radius [optional] Neighbor search radius parameter to use
-    /// HybridSearch. [Recommended ~1.4x voxel size].
+    /// It uses KNN search if only max_nn parameter is provided, Radius search
```

Comment:
Mention `(Not recommended to use on GPU)` next to KNN search, and `(Recommended)` next to HybridSearch. To caution users about the performance of using KNN search on GPU.